README.md

```diff
@@ -1,20 +1,32 @@
 # JQuery/Ajax Quiz
 
-Write your answer below each question. 
+Write your answer below each question.
 
 ### Question 1
-(a) What is a click handler? 
+(a) What is a click handler?
 
-(b) Where in your JS file should you put it, and WHY? 
+A click handler is a jquery function that is set off when you click on whatever jquery
+object that the click handler has been assigned to.
+
+(b) Where in your JS file should you put it, and WHY?
+
+In the $(docuemnt).ready
 
 ### Question 2
-If you get the error `$ is undefined`, what does that mean and what could you do to fix it? 
+If you get the error `$ is undefined`, what does that mean and what could you do to fix it?
+
+It means that the object your working with has not been defined. To fix it you can see if you forgot
+to assign said object to a var, or see if it is in the correct namespace to where it is being called.
 
 ### Question 3
-What should go in the `$(document).ready()` part of your JS file? 
+What should go in the `$(document).ready()` part of your JS file?
+
+Function calls for your jquery code.
```

Comment:
It's unclear what you mean by "function calls for your jquery code." In general, anything that requires an HTML element to be loaded first needs to go in the document ready.